src/Tomcat/Tomcat.ts

```diff
@@ -29,7 +29,7 @@ export class Tomcat {
             const oldServer: TomcatServer[] = this._serverList.splice(index, 1);
             if (oldServer.length > 0) {
                 const catalinaBasePath: string = path.join(this._extensionpath, oldServer[0].getName());
-                Utility.deleteFolderRecursive(catalinaBasePath);
+                fse.remove(catalinaBasePath);
```

Comment:
have you tried if you fse.remove a non-existed folder, what will happen? Is there error to be thrown? If so please keep the original deletFolderRecursive utiltiy method

Reply:
Sure I tried it.